src/pdbpp.py

```diff
@@ -42,6 +42,16 @@ def signature(obj):
             return ' [pip install funcsigs to show the signature]'
 
 
+has_colorama_on_windows = False
+if sys.platform == "win32":
+    try:
+        import colorama
+        has_colorama_on_windows = True
+    except ImportError:
+        # No windows support. Do we want to stop trying all coloring?
+        print("Please install 'colorama' for color support on Windows.")
```

Comment:
This should be done on demand, not during import - at least the printing.

Reply:
Hmm... I'm not sure I fully understand. Are you saying I should put the whole thing (`sys.platform` check and the import) in `__init__` (L341-ish)?

Reply:
Even later - only on demand, when there's actual output.

`import pdb; p = pdb.Pdb()` should not trigger an import of colorama.

Reply:
Looks like the absolute latest that I can import and init `colorama` is:

```diff
@@ -352,6 +341,9 @@ class Pdb(pdb.Pdb, ConfigurableClass, object):
             self._disable_pytest_capture_maybe()
         kwargs = self.config.default_pdb_kwargs.copy()
         kwargs.update(**kwds)
+        if sys.platform == "win32":
+            import colorama
+            colorama.init(strip=False)
         super(Pdb, self).__init__(*args, **kwargs)
         self.prompt = self.config.prompt
         self.display_list = {}  # frame --> (name --> last seen value)
```

If I try importing colorama after that `super()`, then things don't work. I've pushed commit 4bd8cc1ff7f26704661990d1f115144aa8b7a1cb - take a look at let me know what you think.

I know if doesn't satisfy your requirement, but I don't know if there's a way to do so. If there is, it's outside of my skillset.